src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx

```diff
@@ -0,0 +1,176 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+
+import AsyncView from 'app/views/asyncView';
+import Button from 'app/components/button';
+import {PanelItem} from 'app/components/panels';
+
+const STEPS = {
+  tag: 'Tag an error',
+  repo: 'Link to a repo',
+  commit: 'Associate commits',
+  deploy: 'Tell sentry about a deploy',
+};
+
+class ReleaseProgress extends AsyncView {
+  static contextTypes = {
+    organization: PropTypes.object,
+    project: PropTypes.object,
+    router: PropTypes.object,
+  };
+
+  constructor(props, context) {
```

Comment:
not needed?

Reply:
Nothing's being done in the constructor (you don't need to define a constructor to use context).

Reply:
Hmm weird- I must have been doing something real funky in an earlier iteration. Def works without it- thanks!